### PR TITLE
Credential Registration

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- Credentials Registration & Login
- Reset Password

`Note: You can only test using @codeforafrica.org emails for now`

## Type of change

- [x] Chore (non-breaking change which does not add visible functionality but improves code quality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
Conflict @KhadijaMahanga ... and trying to register fails without any indication on the form. Checking console tells me there is an error:

Screenshot from 2022-01-27 13-31-31

**Comment by KhadijaMahanga** [comment]
> Conflict @KhadijaMahanga ... and trying to register fails without any indication on the form. Checking console tells me there is an error:
> 
> Screenshot from 2022-01-27 13-31-31

@kilemensi Oh I see. Let me look into it now

**Review by kilemensi — COMMENTED** [review]
👍🏽 

--

Still stuck on sign up (emails seem to be going into spam folder).

1. URL in the email still appears to point to the BE URL and not the FE one,
Screenshot from 2022-01-27 15-43-47
2. Clicking on it does redirect to FE but I get 👇🏽 error
Screenshot from 2022-01-27 15-45-05
(Could be because this branch is not merged yet & hence we should redirect to this branch URL for reviews?)

File: src/pages/api/admin/config.json [hunk 1]
```diff
@@ -624,6 +577,240 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Account",
```
[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/pages/api/admin/config.json:582]: @kilemensi these are not for storybook. These are for cms structure

**Comment by KhadijaMahanga** [comment]
> 👍🏽
> 
> Still stuck on sign up (emails seem to be going into spam folder).
@kilemensi  I think this could be smtp server issue itself or the dummy email, I am using gmail relay server on localhost and they don't go to spam

> 1. URL in the email still appears to point to the BE URL and not the FE one,
@kilemensi yes, because we were doing redirects instead of updating the url. I have created a [PR](https://github.com/CodeForAfrica/TwoopsTracker/pull/51) here to change that

> 2. Clicking on it does redirect to FE but I get 👇🏽 error
>    (Could be because this branch is not merged yet & hence we should redirect to this branch URL for reviews?)
@kilemensi yes, the branch is not merged. I will update the config var to point to here in the meantime

**Comment by kilemensi** [comment]
> @kilemensi  I think this could be smtp server issue itself or the dummy email, I am using gmail relay server on localhost and they don't go to spam
> 

Cool, we'll check again once we move SES to production @KhadijaMahanga

**Review by nerdlet — APPROVED** [review]
@KhadijaMahanga Looks good! 🚀 My login was successful and was able to verify my email address using `...@codeforafrica.org` email

**Review by kilemensi — APPROVED** [review]
🚀

**Review by gertie-sheshe — APPROVED** [review]

**Pull request merged**